src/pseudopeople/noise.py

```diff
@@ -44,25 +44,31 @@ def noise_form(
     :return:
         Noised form data
     """
-    randomness = get_randomness_stream(form, seed)
+    randomness = get_randomness_stream(form.name, seed)
 
-    noise_configuration = configuration[form.value]
+    noise_configuration = configuration[form.name]
     for noise_type in NOISE_TYPES:
         if isinstance(noise_type, RowNoiseType):
-            # Apply row noise
-            form_data = noise_type(form_data, noise_configuration, randomness)
+            if noise_configuration.to_dict().get("row_noise"):
```

Comment:
Per our conversation this can still throw a Configuration Error if there is a noise type in NOISE_TYPES that doesn't exist on this particular form